ansible/psi/README.md

```diff
@@ -41,4 +41,6 @@ You can run [deploy-all.sh](./deploy-all.sh) to mass-deploy all instances. Exist
 
 All cloud/PSI specific parameters are in [psi_defaults.yml](./psi_defaults.yml), please edit/extend that instead of hardcoding cloud specifics in roles or playbooks.
 
-The instances run our usual [users role](../roles/users/), so if you are in the [ssh-keys](../roles/users/tasks/ssh-keys) list, you can SSH to the instances with `ssh core@10.X.X.X`.
+SSH access
+----------
+The instances run our usual [users role](../roles/users/), so if you are in the [ssh-keys](../roles/users/tasks/ssh-keys) list, you can SSH to the instances. Run [`./openstack-ssh-config`](../openstack-ssh-config) to generate an SSH configuration file from the dynamic inventory, and follow the instructions. Then you can do e. g. `ssh rhos-01-1`.
```

Comment:
I followed this and it generates `/home/mmarusak/.ssh/ansible-openstack.config`. However:
```
 ssh rhos-01-1
Bad owner or permissions on /home/mmarusak/.ssh/ansible-openstack.config
```

```
$ ls -l /home/mmarusak/.ssh/ansible-openstack.config
-rw-rw-r--. 1 mmarusak mmarusak 1985 Aug  2 11:39 /home/mmarusak/.ssh/ansible-openstack.config

```

Reply:
Nice find! Mine is 644, which works. I suppose your umask is "002" (mine is "022"), so new files are group writeable by default.

I updated the commit to remove group/other write permissions.